### DISCUSSION TITLE
Feature Request: add rel="me" attribute to verify website links for Mastodon users

### DISCUSSION THREAD

**Discussion by apreshill** [quarto-dev/quarto-cli#3298] [Feature Requests]
According to the [Mastodon documentation](https://docs.joinmastodon.org/user/profile/#fields):

> Document-based verification and blue ticks are not possible without a central authority. However, Mastodon can cross-reference the links you put on your profile to prove that you are the real owner of those links. In case one of those links is your personal homepage that is known and trusted, it can serve as the next-best-thing to identity verification.

To do this, the link MUST have [a rel="me" attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/me).

See blog post: https://cedricbatailler.me/blog/how-verification-works-on-mastodon/

You can see a PR that adds this to a Hugo theme here: https://github.com/hugo-apero/hugo-apero/pull/93

Outcome:
<img width="325" alt="Screen Shot 2022-11-11 at 9 27 26 PM" src="https://user-images.githubusercontent.com/12160301/201427354-9586b74c-fb88-4589-96b2-5e6f85bcdc6c.png">

**Comment by mcanouil** [comment]
FYI, related to discussion #3207 and the associated issue #3212

**Comment by mcanouil** [comment]
@cscheid just wrote this feature is in v1.3 pre-release. 🥳
See #3333

**Reply by cscheid** [reply]
This is entirely to @effigies's credit, I just merged the PR :)

**Reply by mcanouil** [reply]
Yep, was easier on mobile to link to PR. (username autocomplete only works for involved ones🤷‍♂️).
Thanks @effigies

**Reply by ggirton** [reply]
Looking forward to seeing this in 1.3 -- 1.2 appears to be racking up point updates continually, but maybe this is in there?

**Reply by mcanouil** [reply]
@ggirton the changelog of 1.2 and 1.3 pre-release are on the website near the download section (<https://quarto.org/docs/download/>).

Issue #3212 (and the related PR #3333) is only mentioned as part of 1.3 (_i.e._, no cherry-picking to put it in 1.2).

**Reply by cscheid** [reply]
@ggirton To clarify so other folks aren't confused: the `main` branch will become 1.3 (that's how our development works). 1.2 only gets critical bugfixes and is on a tagged branch `v1.2`.

**Comment by mcanouil** [comment]
Closing this as completed as part of 1.3.
